diagnostics/ENSO_MSE/shared/read_netcdf_2D.py

```diff
@@ -0,0 +1,25 @@
+import numpy as np
+import os.path
+import sys
+
+from scipy.io import netcdf
+
+def read_netcdf_2D(imax, jmax, tmax,  variable,  namein, dataout, undef):

```

Comment:
Jess 
it seems those two are left overs from the previous version, when  was reading binary files instead of netCDF.  Let me merge those two and eliminate imax, jmax, zmax tmax

Reply:
I think I can complete delete them as they are not used in the new code.